lib/version-check.js

```diff
@@ -41,7 +41,7 @@ module.exports = function (gulp, gutil) {
 
                 //On success invoke callback function
                 if (typeof cb !== 'undefined') {
-                    cb();
+                    return cb();
```

Comment:
this is fine for now, but note the guidelines around error handling in node: https://www.joyent.com/developers/node/design/errors

"The usual pattern is that the callback is invoked as callback(err, result), where only one of err and result is non-null, depending on whether the operation succeeded or failed."